components/Favorite/index.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import styled from "styled-components";
+import ProductCard from "../ProductCard";
+import { useContext } from "react";
+import { FavoriteContext } from "../../pages/_app";
+
+export default function Favorite(props) {
+  const context = useContext(FavoriteContext);
+
+  return (
+    <StyledFavorite>
```

Comment:
I would suggest a more precise name and the `role` attribute because you have set `list-style` to `none` which is an issue in Safari.
```suggestion
    <StyledFavoriteList role="list">
```